packages/base-styles/_z-index.scss

```diff
@@ -51,6 +51,7 @@ $z-layers: (
 
 	// The draggable element should show up above the entire UI
 	".components-draggable__clone": 1000000000,
+	".block-editor-block-draggable-chip-wrapper": 1000000000,
```

Comment:
It might be worth using a decimal literal for readability `1_000_000_000`

Not sure how important this is, but it looks like the parent z-index stacking context for the chip wrapper is `.interface-interface-skeleton__content`. This means that despite the high z-index here items in sidebar and actions will be on top of the chip from natural stacking order.

https://user-images.githubusercontent.com/1270189/135175956-8d229de1-cd44-44c7-b152-f32cd93781c6.mp4

<img width="1067" alt="CleanShot 2021-09-28 at 15 43 55@2x" src="https://user-images.githubusercontent.com/1270189/135175920-d0534887-133f-4298-b099-1e827a1e9a6d.png">